go/lib/header/header.go

```diff
@@ -0,0 +1,6 @@
+package header
+
+const (
+	ContentType = "Content-Type"
```

Comment:
You could use https://github.com/go-http-utils/headers for these headers probably.